ddl/ddl.go

```diff
@@ -732,7 +736,7 @@ func (d *ddl) buildIndex(ctx context.Context, t table.Table, idxInfo *model.Inde
 			vals = append(vals, val)
 		}
 		// build index
-		kvX := kv.NewKVIndex(t.IndexPrefix(), idxInfo.Name.L, unique)
+		kvX := kv.NewKVIndex(t.IndexPrefix(), idxInfo.Name.L, idxInfo.ID, unique)
```

Comment:
for error log later maybe, like ErrKeyExist, we can returns the index name and index value in error for debug easily.